src/desktop/DesktopMain.js

```diff
@@ -82,6 +82,17 @@ if (process.argv.indexOf("-r") !== -1) {
 				wm.getAll().forEach(w => w.show())
 			}
 			handleArgv(args)
+		}).on('open-url', (e, url) => {
+			// MacOS mailto handling
+			e.preventDefault()
+			if (!url.startsWith('mailto:')) {
+			} else if (app.isReady()) {
```

Comment:
Can we always do it with `once`?

Reply:
if we attach the 'open-url' event handler with `once` we have to reattach it right away if we want to handle any more clicks on mailto links, with `on` it would remain attached?